source/connect/favorite-connections/import-export-ui/export.txt

```diff
@@ -0,0 +1,113 @@
+.. _compass-export-connections-ui:
+
+==============================================
+Export Favorite Connections in |compass-short|
+==============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can export favorite connections in |compass|. Colleagues can import your 
+favorite connections, or you can use them in your other workspaces.
+
+About This Task
+---------------
+
+.. include:: /includes/fact-export-favorite-connection.rst
+
+Procedure
+---------
+
+To export encrypted favorite connections with |compass-short|:
+
+.. procedure::
+    :style: normal
+    
+    .. step:: On the Connect screen, hover over :guilabel:`Saved Connections` 
+        in the left-side navigation and click the :guilabel:`Ellipses (...)` 
+        button.
+
+    .. step:: Select :guilabel:`Export saved connections`. 
+
+    .. step:: In the :guilabel:`Export saved connections` dialog box, select the 
+        connection names you want to export. 
+
+    .. step:: Under :guilabel:`Target File`, click :guilabel:`Select a file...` 
+        to specify the output file destination.
+
+    .. step:: *Optional*. Toggle :guilabel:`Remove secrets` to omit passwords, 
+        access tokens, and other sensitive information from the exported file.
+
+        If you select :guilabel:`Remove secrets`, you cannot specify an 
+        encryption password.
+    
+    .. step:: *Optional*. Specify an :guilabel:`Encryption Password` to encrypt 
+        passwords, access tokens, and other sensitive information.
+        
+        If you do not encrypt the export file, database usernames and passwords 
+        are exported in plaintext. Only export unencrypted connections if no 
```

Comment:
Usernames are always exported in plaintext here

Reply:
Updated.